content/en/docs/Extensions/httpFormPost.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "HTTP form POST"
+linkTitle: "HTTP form POST"
+OpenSubsonic:
+  - Extension
+description: >
+  Add support for POST request to the API (application/x-www-form-urlencoded)
+---
+
+**OpenSubsonic version**: [1](../../opensubsonic-versions)
+
+**OpenSubsonic extension name** `supportFormPost` (As returned by [`getOpenSubsonicExtensions`](../../endpoints/getopensubsonicextensions))
```

Comment:
i think the `support` is redundant in the name. for example we don't have `supportTranscodeOffset` how about just `formPost`?